web/beacon-app/src/features/tenants/components/TenantTable.tsx

```diff
@@ -29,10 +29,22 @@ export default function TenantTable() {
       <Table
         columns={[
           { Header: 'Tenant Name', accessor: 'name' },
-          { Header: 'Environment Label', accessor: 'env' },
+          { Header: 'Environment Label', accessor: 'environment_type' },
```

Comment:
The environment_type has a lowercase `d`. This can be easily fixed on the backend and I can do that when working on a BE story this week.